src/effects/VST/VSTEffect.cpp

```diff
@@ -1104,22 +1104,15 @@ void VSTEffectMessage::Merge(Message && src)
 
 std::unique_ptr<EffectInstance::Message> VSTEffectInstance::MakeMessage() const
 {
+   // The purpose here is just to allocate vectors (chunk and paramVector)
+   // with sufficient size, not to get the values too
    VSTEffectSettings settings;
-   FetchSettings(settings, false);
+   FetchSettings(settings, /* doFetch = */ false);
 
    VSTEffectMessage::ParamVector paramVector;
    paramVector.resize(settings.mParamsMap.size(), std::nullopt);
 
-   ForEachParameter
-   (
-      [&](const ParameterInfo& pi)
-      {
-         paramVector[pi.mID] = settings.mParamsMap[pi.mName];
-         return true;
-      }
-   );
```

Comment:
As I explained, this is still incorrect:  don't just delete, but replace with a sizing of paramVector to `mAEffect->numParams` and filled with nullopts.